src/style.css

```diff
@@ -0,0 +1,130 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+}
+
+.container {
+  width: 200px;
+  height: 406px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 0 auto;
+  padding-top: 32px;
+}
+
+.img {
+  margin: 0 auto;
+
+  max-width: 160px;
+  height: 134px;
+}
+
+.container__2 {
+  padding: 0 16px;
+}
+
+.description {
+  padding-top: 40px;
+  padding-bottom: 4px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.product-code {
+  padding-bottom: 16px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.block-stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--4 .stars__star:nth-child(n+5) {
+  background-image: url(./images/star.svg);
+}
+
+.reviews {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.block-price {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-bottom: 16px;
+}
+
+.price {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.sum {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.button {
+  width: 166px;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.button__buy {
+  display: flex;
+  justify-content: center;
+  padding-top: 10px;
+
+  text-decoration: none;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+}
+
+.button__buy:hover {
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  height: 40px;
+}
+
+.button:hover {
```

Comment:
Hover on .button makes the text invisible! Please fix
![image](https://user-images.githubusercontent.com/48877776/215093392-b71e2e04-dcf5-4a9c-82e3-525875083c74.png)